src/Pages/Homepage.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import { createStyles, makeStyles, Theme } from '@material-ui/core/styles'
+
+import { BlueBoxes } from '../Components'
+import Flag from '../Assets/Logos/flag_bon_voyage.svg'
+
+const useStyles = createStyles(
+  makeStyles(({ palette, breakpoints }: Theme) => ({
+    container: {
+      [breakpoints.up('md')]: {
+        paddingLeft: '200px',
+        paddingRight: '200px',
+      },
+      [breakpoints.up('xs')]: {
```

Comment:
ça devrait pas être `breakpoints.up('sm')` ici comme tu définis un only xs en dessous ? parce que breakpoints.up c'est supérieur ou égal
```suggestion
      [breakpoints.up('sm')]: {
```